ui/lib/config-ui/addon/components/messages/page/list.js

```diff
@@ -136,17 +103,23 @@ export default class MessagesList extends Component {
     }
   }
 
-  @action
-  onFilterInputChange(pageFilter) {
-    this.transitionToMessagesWithParams({ pageFilter });
+  @task
+  *handleSearch(evt) {
+    evt.preventDefault();
+    const formData = new FormData(evt.target);
+    // shows loader to indicate that the search was executed
+    yield timeout(Ember.testing ? 0 : 250);
+    const params = {};
+    for (const key of formData.keys()) {
+      const val = formData.get(key) || undefined;
+      params[key] = val;
+    }
+    this.transitionToMessagesWithParams(params);
   }
 
   @action
-  onFilterChange(filterType, [filterOption]) {
-    const param = {};
-    param[filterType] = filterOption;
-    param.page = 1;
-    this.transitionToMessagesWithParams(param);
+  resetFilters() {
+    this.transitionToMessagesWithParams({ pageFilter: undefined, status: undefined, type: undefined });
```

Comment:
For consistency I wonder if clicking reset should match the defaults defined in the [controller](https://github.com/hashicorp/vault/blob/main/ui/lib/config-ui/addon/controllers/messages/index.js#L10-L14)? 
```
  pageFilter = '';
  type = null;
  status = null;
```